### ISSUE TITLE
Call recorder

### ISSUE THREAD

**Issue by lbdroid** [GrapheneOS/os-issue-tracker#1335]
The previous issue #868 was unceremoniously deleted right after a clear misunderstanding;
Please take the time to understand what is being suggested before dismissing it as "not an option" or "insecure" or "OMG ROOT".
I promise that my suggestion DOES NOT IN ANY WAY require root.

The suggestion: (@lbdroid)
=====================
That's very frustrating to hear. This is a very important feature that is available in factory google builds for these devices, but lost when switching to graphene.

How about an alternative that doesn't even require modification of existing packages?
https://github.com/chenxiaolong/BCR

The main problem with external call recording applications is that they require some form of elevated privileges in order to be able to record the phone call audio streams. The above works quite well, but in order to gain those elevated privileges, it has to be packaged as a "magisk module", which mounts it as a privileged system application (/system/priv-app/...), which is strictly a non-starter for GrapheneOS.

On the other hand, if it was built in as a system application, the dependency on "magisk" would be removed altogether.

The response: (@thestinger)
======================
Completely not an option. It's insecure. We are not giving root access to an application.

Stop bumping the issue.

The PROBLEM with the response:
===========================
Nothing in the suggestion recommended giving "root access" to anything. In fact, the suggestion specifically discusses how to accomplish this WITHOUT giving root access to anything.

As things stand right now, THE ONLY WAY to record calls is with root. ****THIS IS NOT ACCEPTABLE****, hence the need for a solution that DOES NOT require root, which is why I provided an option that DOES NOT require root.

To put it another way, how many users of GrapheneOS do you want to push into using magisk/root in order to be able to do something that they should be able to do withOUT root?

**Comment by thestinger** [comment]
@lbdroid I told you to stop. This is the last warning you're ever getting.

Call recording is no longer planned because of the behavior of people bothering us about it.
